File: api/src/main/java/com/joey/mysite/jdbc/bean/BcCard.java
package com.joey.mysite.jdbc.bean;

public class BcCard {
    private String id;

    private String name;

    private String userId;

    private String cardTypeId;

    private String balance;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id == null ? null : id.trim();
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name == null ? null : name.trim();
    }

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId == null ? null : userId.trim();
    }

    public String getCardTypeId() {
        return cardTypeId;
    }

    public void setCardTypeId(String cardTypeId) {
        this.cardTypeId = cardTypeId == null ? null : cardTypeId.trim();
    }

    public String getBalance() {
        return balance;
    }

    public void setBalance(String balance) {
        this.balance = balance == null ? null : balance.trim();
    }
}

File: bill.api/app/database/entity/BcCard.ts
import {Column,Entity} from "../decorator";
import {BaseEntity} from "../BaseEntity";

@Entity()
export class BcCard extends BaseEntity {
    @Column()
    name: string;

    @Column({ type: "double"})
    balance: number;

    @Column({length: 36})
    userId: string | null;

    @Column({length: 36})
    cardTypeId: string | null;
}
